src/worker/core/draft/genPlayersWithoutSaving.ts

```diff
@@ -57,12 +58,15 @@ const genPlayersWithoutSaving = async (
 	// Do one season at a time, keeping the lowest pot players in college for another season
 	let enteringDraft: typeof remaining = [];
 
-	for (let i = 0; i < 4; i++) {
-		let cutoff = 0; // For football, only juniors and seniors
+	for (let i = 0; i < draftAge[1] + 1; i++) {
+		let cutoff = draftAge[1] - draftAge[0]; // For football, only juniors and seniors by default.
```

Comment:
I think this should still be 0, otherwise in FBGM it will take the top `draftAge[1] - draftAge[0]` players and have them declare for the draft, until the condition below triggers.